src/ucan.ts

```diff
@@ -11,53 +11,103 @@ export * from "./crypto.js"
 
 export type { MultihashDigest, MultibaseEncoder, MultihashHasher }
 
+/** Arbitrary (service / capability specific) information included in a UCAN {@link Body} */
 export type Fact = Record<string, unknown>
 
+/**
+ * A participant in the UCAN system that can be identified by {@link DID}.
+ */
 export interface Identity {
   did(): DID
 }
 
+/**
+ * The intended recipient of a given UCAN.
+ */
 export interface Audience extends Identity {}
 
+/**
+ * An {@link Identity} that can verify signatures produced with the algorithm `A` (see {@link Verifier}).
+ */
 export interface Authority<A extends number = number>
   extends Identity,
     Verifier<A> {}
 
+/** The {@link Identity} that created and signed a given UCAN. */
```

Comment:
It's less about it been signing identity, and more about that it been identity that can sign. For example when receiving a UCAN it's issuer will be represented via `Authority` not the `Issuer` as later requires access to private key. On the other hand when issuing or delegating UCAN you need to provide `Issuer` in order to be able to sign.

With this said, do you think this would describe it better ?

```suggestion
/** The {@link Identity} that can issue / delegate UCAN by signing */
```